types/src/channels/base.ts

```diff
@@ -0,0 +1,27 @@
+// deno-lint-ignore-file camelcase
+
+// https://discord.com/developers/docs/resources/channel#channel-object-channel-types
```

Comment:
I think it'd be better to use JSDoc's `@link` attribute for these links, what do you think? Also maybe we could add descriptions for fields based on the API docs?

Reply:
yeah definitely